.devcontainer/post.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ -f install.R ]
+then
+    echo "installing R packages..."
+    Rscript install.R
```

Comment:
This script could be an Rscript script (if you want to call `cat(....)` etc).  Moreover there is no gain in farming out to yet another script.  Just do `Rscript -e 'install.packages(c("whatever", "you", "put", "here"))'`.

My $0.02 but not my space.  Still, this is a _r2u_ codespace so I would tend to keep the R admin lean and simple to let r2u shine.

Reply:
I actually did not want the user to edit the stuff in `.devcontainer`, thus the bifurcation into a wrapper script (always there, no need for the user to edit, no Linux knowledge necessary), and an optional "pure R" script in the (visible) main directory.